features/features.go

```diff
@@ -67,6 +67,11 @@ const (
 	// of "orphaned" certs we have. However, it also requires clients to properly
 	// implement polling the Order object to wait for the cert URL to appear.
 	AsyncFinalize
+	// SetCommonName causes the certificates issued to omit the commonName field
```

Comment:
This description is for the old `OmitCommonName` and should be altered.

Reply:
Good catch, thanks!